backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/SyncToRemoteStatus.java

```diff
@@ -0,0 +1,10 @@
+package com.virtuslab.gitmachete.backend.api;
+
+public interface SyncToRemoteStatus {
+  enum Status {
```

Comment:
This naming is a bit awkward coz "status" now means both the `Status` value itself, and the pair (Status, remote name)... no idea as for the moment how could either of them be renamed though :/

Reply:
Maybe `Status` could be called more specifically `Relation` instead?